cmd/serve/dlna/cds.go

```diff
@@ -130,6 +130,20 @@ func (cds *contentDirectoryService) readContainer(o object, host string) (ret []
 		return
 	}
 
+	// if there's a "Subs" child directory, add its children to the list as well,
+	// so mediaWithResources is able to find them.
+	for _, node := range dirEntries {
+		if node.Name() == "Subs" && node.IsDir() {
```

Comment:
This should probably be a case insensitive compare so `strings.EqualFold`

Reply:
Ok, will do.

Reply:
Done